npm/vite-dev-server/src/index.ts

```diff
@@ -14,3 +15,11 @@ export async function startDevServer (startDevServerArgs: StartDevServerOptions)
 
   return { port, close: app.httpServer!.close }
 }
+
+export function devServer (cypressDevServerConfig: Cypress.DevServerConfig, devServerConfig?: Omit<InlineConfig, 'base' | 'root'>) {
```

Comment:
Instead of duplicating `Omit<InlineConfig, 'base' | 'root'>` in two places, can you make a type or interface like `CypressViteDevServerConfig`?